doc/guides/examples.md

```diff
@@ -94,3 +94,16 @@ query hello {
 The query above will only work if the authenticated user has the `admin` role. You can see this rule being applied with the `@hasRole(role: "admin")` directive in [examples/keycloak/server.js](../../examples/keycloak/server.js#L22).
 
 Try change the the role to a made up role and restart the server. Try the sample query again and verify that an error is displayed.
+
+## 3rd party integrations example
+
+`integrations/index.js` shows, how it is easy to external web service services under a GraphQL based API.
+
+```
+$ node examples/integrations/index.js
+🚀 Server ready at http://localhost:4000/graphql
+```
+
+Open http://localhost:4000/graphql and you can run already prepared query there.
+
+Columns that are directly in `getCharacterInfo` are resolved from [Star Wars People API](https://swapi.co/).
```

Comment:
@wojta since most of the integration will be in the resolvers do you think it might make sense to document a simple code example here? Something very easy like:
The suggested pattern for integrating with a REST API is a simple flow as follows:
```
const resolvers = {
  Query: {
    queryREST: async (obj, args, context) => {
       const result = axios.get(YOUR_ENDPOINT_HERE, params);
       return result;
    }
  }
}
```
for a working example, please see {FILE} etc...

Not sure about using a library name like axios in the docs but we could just say you can use any http library you like.

Reply:
Yes. Yes. Yes Let's not mention API we integrate here and build generic example.

Reply:
Ok, I did it somehow. Linked example's source lines.